languages/csharp/exercises/concept/enums/.meta/Example.cs

```diff
@@ -2,20 +2,37 @@
 
 public enum LogLevel
 {
-    Unknown = 0,
-    Info = 1,
-    Warning = 2,
-    Error = 4
+    Trace = 0,
+    Debug = 1,
+    Info = 4,
+    Warning = 5,
+	Error = 6,
+	Fatal = 7,
+	Unknown = 42
 }
 
 public static class LogLine
 {
-    public static LogLevel ParseLogLevel(string logLine) =>
-        Enum.TryParse<LogLevel>(GetLogLevel(logLine), true, out LogLevel logLevel) ? logLevel : LogLevel.Unknown;
+    public static LogLevel ParseLogLevel(string logLine) {
```

Comment:
This is exactly what I meant. Nice job.

Reply:
Thanks man! I'll go over your suggestions.